backend/compact-connect/stacks/persistent_stack/provider_table.py

```diff
@@ -51,6 +52,21 @@ def __init__(
             sort_key=Attribute(name='licenseGSISK', type=AttributeType.STRING),
             projection_type=ProjectionType.ALL,
         )
+        self.add_global_secondary_index(
+            index_name=self.compact_transaction_gsi_name,
```

Comment:
Do we need a DataMigration to add this GSI field to existing transactions in the table?